.github/workflows/CI.yml

```diff
@@ -7,6 +7,42 @@ on:
 
 
 jobs:
+  lint:
+    name: Linting
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        php:
+          - '8.1'
+          - '8.2'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Lint PHP
+        run: Build/Scripts/runTests.sh -p ${{ matrix.php }} -s lint
+
+  code-quality:
+    name: Code Quality
+    runs-on: ubuntu-22.04
+    env:
+      php: '8.1'
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install testing system
```

Comment:
composer update / validate / normalize is not needed for PHP checks, so they should be removed. Also in the runTests.sh and docker-compose.yaml.